### ISSUE TITLE
[Auth Emulation] App doesn't show mocking UI

### ISSUE THREAD

**Issue by craigshoemaker** [Azure/static-web-apps-cli#92]
When I try to navigate to the auth emulation page that mocks a user account, the browser stays on the app and never renders the auth mocking UI.

## Steps to reproduce
1. Run `swa start ./public`
2. Navigate to the app in the browser at _http://127.0.0.1:4200/_ (to verify the local server is working)
3. Navigate to _http://127.0.0.1:4200/.auth/login/github_ to attempt to emulate a login

## Expected behavior
The app should display the auth mocking UI.

## Environment
- **OS:** Windows 10 Enterprise
- **Browser:** Chrome
- **Version:** 88.0.4324.190 (Official Build) (64-bit)

## Logs
```
$ swa start ./public
INFO: Found SWA configuration file: C:\Users\cshoe\Documents\data\code\sandbox\swa-cli-demo\.github\workflows\azure-static-web-apps-ambitious-sand-004598a1e.yml
>> detected runtime: node.js
[auth] Auth server listening on http://0.0.0.0:4242
[ swa] SWA listening on http://0.0.0.0:4280        
[ app] Starting up http-server, serving C:\Users\cshoe\Documents\data\code\sandbox\swa-cli-demo\public
[ app] Available on:
[ app]   http://192.168.1.2:4200
[ app]   http://100.64.76.144:4200
[ app]   http://127.0.0.1:4200
[ app]   http://172.30.240.1:4200
[ app] Unhandled requests will be served from: http://0.0.0.0:4280/?
[ app] Hit CTRL-C to stop the server
[ app] [2021-02-25T14:44:36.621Z]  "GET /" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.190 Safari/537.36"
[ app] (node:19428) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated
[ app] [2021-02-25T14:44:37.299Z]  "GET /global.css" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.190 Safari/537.36"
[ app] [2021-02-25T14:44:37.300Z]  "GET /build/bundle.css" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.190 Safari/537.36"
[ app] [2021-02-25T14:44:38.038Z]  "GET /build/bundle.js" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.190 Safari/537.36"
[ app] [2021-02-25T14:44:38.865Z]  "GET /favicon.png" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.190 Safari/537.36"
[ app] [2021-02-25T14:44:58.896Z]  "GET /.auth/login/github" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, 
like Gecko) Chrome/88.0.4324.190 Safari/537.36"
[ swa] proxy> GET 0.0.0.0:4280/?/.auth/login/github
```

**Comment by manekinekko** [comment]
Hey @craigshoemaker It looks like you tried to access the wrong port. I am sorry about the confusing. 
The URL should be http://127.0.0.1:4280/.auth/login/github (port `4280`).

![image](https://user-images.githubusercontent.com/1699357/109186785-9218e380-7791-11eb-9db1-eeba0916b081.png)

@anthonychu any idea how can we avoid this from happening again?

**Comment by craigshoemaker** [comment]
Thanks, @manekinekko!

I'll submit a PR to the readme to help call it out a bit more explicitly.

**Comment by manekinekko** [comment]
PR merged in 6a837dbbe1d41bf00f2ff0d7509cd614c74374f6

**Comment by anthonychu** [comment]
Opened #94 to address
